gnx.js

```diff
@@ -405,6 +412,8 @@ const buildMutation = function (name) {
 
     if (type.endpoint) {
       const argsObject = { input: { type: new GraphQLNonNull(type.inputType) } }
+      const argsDeleteObject = { id: { type: new GraphQLNonNull(GraphQLID) } }
```

Comment:
since these args are only used inside the delete field object, we could just declare them there